src/binary_helpers.c

```diff
@@ -41,11 +41,13 @@ SR_PRIV int bv_get_value(float *out, const struct binary_value_spec *spec,
 	VALUE_TYPE(BVT_UINT8, read_u8, sizeof(uint8_t));
 
 	VALUE_TYPE(BVT_BE_UINT16, read_u16be, sizeof(uint16_t));
+	VALUE_TYPE(BVT_BE_UINT24, read_u24be, 3);
```

Comment:
Honestly not sure, but would it be more in keeping to make these 3 * sizeof(uint8_t) ?